### PR TITLE
Migrate tests from "cmd/mattermost/commands/command_test.go" to use testify

### PR DESCRIPTION
**Pull request by phillipahereza**
#### Summary
Migrate tests from `cmd/mattermost/commands/commands_test.go` to use testify

#### Ticket Link

[Jira Ticket](https://mattermost.atlassian.net/browse/MM-19010)

Fixes https://github.com/mattermost/mattermost-server/issues/12413

### CODE REVIEW

File: cmd/mattermost/commands/command_test.go [hunk 1]
```diff
@@ -117,11 +117,11 @@ func TestCreateCommand(t *testing.T) {
...
-				if len(cmds) == 0 || cmds[0].Trigger != "testcmd" {
-					t.Fatal("Failed to create command")
-				}
+				require.NotEqual(t, len(cmds), 0, "Failed to create command")
+				require.Equal(t, cmds[0].Trigger, "testcmd", "Failed to create command")
 				assert.Contains(t, string(actual), "PASS")
 			} else {
+
 				if len(cmds) > 1 {
 					t.Fatal("Created command that shouldn't have been created")
```
Comment (hanzei) [inline cmd/mattermost/commands/command_test.go]: You missed this one :wink:

File: cmd/mattermost/commands/command_test.go [hunk 2]
```diff
@@ -117,11 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
 
 			if testCase.ExpectedErr == "" {
-				if len(cmds) == 0 || cmds[0].Trigger != "testcmd" {
-					t.Fatal("Failed to create command")
-				}
+				require.NotEqual(t, len(cmds), 0, "Failed to create command")
```
Comment (hanzei) [inline cmd/mattermost/commands/command_test.go]: You can use  `require.Len` here.

File: cmd/mattermost/commands/command_test.go [hunk 3]
```diff
@@ -117,11 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
```
Comment (hanzei) [inline cmd/mattermost/commands/command_test.go]: A bit out of scope, but would you also please check the returned `error` here?

File: cmd/mattermost/commands/command_test.go (hunk 2 above)
Comment (phillipahereza) [reply cmd/mattermost/commands/command_test.go]: @hanzei doesn't `require.Len` test if the length is equal to some number but in this case, we're testing if the length is not equal to 0

Comment (hanzei) [reply cmd/mattermost/commands/command_test.go]: I see. I thought we had expectations on length of the list, but it seams like we don't

File: cmd/mattermost/commands/command_test.go (hunk 3 above)
Comment (phillipahereza) [reply cmd/mattermost/commands/command_test.go]: @hanzei also `ListCommands` returns `([]*Command, *Response)` and not an error

Comment (hanzei) [reply cmd/mattermost/commands/command_test.go]: I mean checking the `Error` field of `model.Response`.

File: cmd/mattermost/commands/command_test.go (hunk 2 above)
Comment (streamer45) [reply cmd/mattermost/commands/command_test.go]: We could probably use `require.NotEmpty`.

**Comment by hanzei** [comment]
@phillipahereza Let me know, if you have any questions about my feedback

**Comment by phillipahereza** [comment]
> @phillipahereza Let me know, if you have any questions about my feedback


@hanzei I think I implemented the changes you requested

**Comment by hanzei** [comment]
Hey @phillipahereza,

It seams like you already migrated `cmd/mattermost/commands/command_test.go`with https://github.com/mattermost/mattermost-server/pull/12421. :wink: I think you can close this PR.

**Comment by hanzei** [comment]
Closing this as the code is already migrated.

**Pull request closed without merging**